phpunit/phpunit/10.0/tests/bootstrap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require dirname(__DIR__).'/vendor/autoload.php';
+
+if (file_exists(dirname(__DIR__).'/config/bootstrap.php')) {
```

Comment:
I think this is an unnecessary check

Reply:
@a1812 it is required to be compatible with Symfony 4.4 because also there people can update there recipes to latest version: https://github.com/symfony/recipes/blob/main/symfony/console/4.4/config/bootstrap.php